ietf/templates/includes/footer-iab.html

```diff
@@ -0,0 +1,35 @@
+<div 
+    style="
+        background-image: url('static/img/background-dark.jpg');
+        height: 48px;
```

Comment:
The background-image as an inline style I get (easier than dealing with assets in CSS). But maybe the height could be moved to a CSS file?

Reply:
changed